chunk 1: document .St -p1003.1-2024 (upstream commit ff948a62)
chunk 2: base paragraph indentation 5n (upstream commits 5d2e49f8, ab9e82cd)

Index: tmac/groff_mdoc.7.man
--- tmac/groff_mdoc.7.man.orig
+++ tmac/groff_mdoc.7.man
@@ -2514,6 +2514,8 @@ Part 1: System API
 .St -p1003.1-2004
 .It Li \-p1003.1\-2008
 .St -p1003.1-2008
+.It Li \-p1003.1\-2024
+.St -p1003.1-2024
 .El
 .Pp
 .
@@ -5088,17 +5090,16 @@ register to zero.
 .Pp
 The paragraph and subsection heading indentation amounts can be changed
 by setting the registers
-.Ql IN
+.Ql BP
 and
 .Ql SN .
 .
-.Dl groff \-Tutf8 \-rIN=5n \-rSN=2n \-mdoc foo.man | less \-R
+.Dl groff \-Tutf8 \-rBP=4n \-rSN=2n \-mdoc foo.man | less \-R
 .
-The default paragraph indentation is 7.2n on typesetters and 7n on
-terminals.
+The default paragraph and subsection heading indentation amounts are 5n
+and 3n, respectively.
 .
-The default subsection heading indentation amount is 3n;
-section headings are set with an indentation of zero.
+Section headings are set with an indentation of zero.
 .
 .
 .Pp
